provisioner/k8s/main.go

```diff
@@ -56,9 +56,11 @@ func main() {
 	var enableLeaderElection bool
 	var probeAddr string
 	var systemNamespace string
+	var unpackImage string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.StringVar(&systemNamespace, "system-namespace", "rukpak-system", "Configures the namespace that gets used to deploy system resources")
+	flag.StringVar(&systemNamespace, "system-namespace", "rukpak-system", "Configures the namespace that gets used to deploy system resources.")
+	flag.StringVar(&unpackImage, "unpack-image", "quay.io/tflannag/unpack:latest", "Configures the container image that gets used to unpack Bundle contents.")
```

Comment:
Note: we'll need to create a dedicated container image in the operator-framework quay organization for the bundle unpacking component.